tests/test_memnn.py

```diff
@@ -73,6 +74,7 @@ def test_labelcands_multi(self):
         self.assertGreater(valid['hits@1'], 0.95)
         self.assertGreater(test['hits@1'], 0.95)
 
+    @unittest.skip
```

Comment:
should we just delete the test if we're skipping it?

Reply:
whoops!